final-project/sqlite-schemas.txt

```diff
@@ -0,0 +1,60 @@
+CREATE TRIGGER add_research_title AFTER INSERT ON research FOR EACH ROW BEGIN INSERT INTO research (title) VALUES (research.link); END;
```

Comment:
Why is this checked in?

If it's for future documentation, might I recommend Markdown?

Reply:
I never thought to stick it in my own files. I'll use markdown for documentation now.

Why this is checked in... umm.. laziness on my part? :

This is a helper document for me. I kept having to write and re-write these SQL commands to debug the database. My plan was to delete this file when I am done with this section of the code. However, maybe there is a better way to keep track of this.

Reply:
The other thing is, the SQLite databases are not that flexible, I cannot rename, remove, or edit data types in columns once they are created. If I want to make any major changes, I have to drop the table then repopulate it. The queries in this document are easy to paste in to repopulate the database with some initial products and research. 

Probably better for me to keep these on my local branch. Like you mentioned earlier, if I was working with other developers I would never check-in a file like this.

Reply:
Hmmm, seems like you need migrations for those cases, not re-creations of new DBs